ci-operator/step-registry/ipi/alibabacloud/post/ipi-alibabacloud-post-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: ipi-alibabacloud-post
+  steps:
+  - ref: gather-alibabacloud-console
```

Comment:
Did you want our usual [`chain: gather`][1] in here?

[1]: https://steps.ci.openshift.org/chain/gather

Reply:
Probably? I modeled this after the AWS version. What would this look like with the usual?